### PR TITLE
Add user creation command and associated files

### PR DESCRIPTION
**Pull request by jchristgit**

### CODE REVIEW

**Comment by jchristgit** [comment]
The linter is WRONG! I know this because I wrote it.

**Comment by Xithrius** [comment]
skill issue

File: k8s/rbac/create-user.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  - client auth
+EOF
+
+kubectl certificate approve "$USERNAME"
+kubectl get csr "$USERNAME" -o jsonpath='{.status.certificate}'| base64 -d > "$FILENAMEBASE.crt"
+
+echo "Send the following files to the minion:"
+echo
+echo "  $USERNAME.crt"
+echo "  $USERNAME.key"
```
Comment (GDWR) [inline k8s/rbac/create-user.sh:43]: this output 
```
  pydis:leroy.jenkins.crt
  pydis:leroy.jenkins.key
```

when the file names are 
```
  leroy.jenkins.crt
  leroy.jenkins.key
```

**Comment by ChrisLovering** [comment]
"I think we can close it" - Johannes 2023

**Pull request closed without merging**